frontend/src/components/TaskHandler.tsx

```diff
@@ -0,0 +1,116 @@
+import React, {ChangeEvent, useState} from "react";
+import './TaskHandler.scss';
+import {ITodo} from "../models/Todo";
+import TodoTask from "./TodoTask";
+
+
+export default function TaskHandler() {
```

Comment:
Here is a lot of Code which isnt finished. We dont want any not functional or not working components in main